nums/numpy/api/creation.py

```diff
@@ -1116,3 +1116,77 @@ def zeros_like(prototype, dtype=None, order="K", shape=None):
     if order is not None and order != "K":
         raise NotImplementedError("Only order='K' is supported.")
     return zeros(shape, dtype)
+
+
+def asarray(a, dtype=None, order=None, *, like=None):
+    """Convert the input to an array.
+
+    Parameters
+    ----------
+    a : array_like
+        Input data, in any form that can be converted to an array.  This
+        includes lists, lists of tuples, tuples, tuples of tuples, tuples
+        of lists and ndarrays.
+    dtype : data-type, optional
+        By default, the data-type is inferred from the input data.
+
+    Returns
+    -------
+    out : ndarray
+        Array interpretation of `a`.  No copy is performed if the input
+        is already an ndarray with matching dtype and order.  If `a` is a
+        subclass of ndarray, a base class ndarray is returned.
+
+    Raises
+    ------
+    NotImplementedError
+        If you pass a non-`None` value to `order` or `like`.
+
+    See Also
+    --------
+    asanyarray : Similar function which passes through subclasses.
```

Comment:
Delete other APIs we don't officially support.

Reply:
I deleted the two that we'll never support. The others we'll conceivably implement in the future. Do you think I should remove them now and add them back later, or just keep them?

Reply:
I figure if we delete them now we'll probably forget to add it back later

Reply:
If you want to implement them, then sure leave them behind. But I believe they'll link to NumPy docs, so make sure to recompile locally to make sure

Reply:
Okay, sounds good. I'll compile the documentation and check